_layouts/figure.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <link rel="stylesheet" type="text/css" href="https://d3fvh0lm0eshry.cloudfront.net/static/css/newamericadotorg.lite.css"/>
```

Comment:
Should I be including this same css in the main site?

Reply:
You could definitely switch from their base CSS to our base CSS, which I would do by using these two stylesheets instead of the 18F-based ones, and adjusting from their. Our base styles aren't in amazing shape, but they're probably an easier base for our branding.

Reply:
I would also note that I think you can grab a number of static assets from the website and they should work fine, e.g., I'm pretty sure you can just use the same version of the logo that newamerica.org links to.

Reply:
We've got a new UX person (@outstandy) that can support with moving in this CSS